EIPS/eip-4762.md

```diff
@@ -251,6 +236,13 @@ When a **write** event of `(address, sub_key, leaf_key)` occurs, perform the fol
 
 Note that tree keys can no longer be emptied: only the values `0...2**256-1` can be written to a tree key, and 0 is distinct from `None`. Once a tree key is changed from `None` to not-`None`, it can never go back to `None`.
 
+Note that values should only be added to the witness if there is sufficient gas to cover their associated event costs.
+
+`CREATE*` and `*CALL` reserve 1/64th of the gas before the nested execution. In order to match the behavior of this charge with the pre-fork behavior of access lists: 
+
+ * this 1/64th of the gas is subtracted **AFTER** charging the witness costs when performing a `CALL`, `CODECALL`, `DELEGATECALL` or`STATICCALL`
```

Comment:
Nit: in the case of `*CALL` (compared to `CREATE*`) it isn't a strict reservation, but a "cap check" of the max amount of gas you can use. 

If the `*CALL` parameter provided a `gas` parameter that is less 63/64, then _that_ `gas` is reserved.

```suggestion
 * this minimum 1/64th gas reservation is checked **AFTER** charging the witness costs when performing a `CALL`, `CODECALL`, `DELEGATECALL` or`STATICCALL`
```

Anyone reading the spec as it's today will understand the important clarification being made here. So I'm fine if we dismiss this nit.